package.json

```diff
@@ -15,7 +15,7 @@
   "author": "Triple",
   "license": "Apache-2.0",
   "dependencies": {
-    "hls.js": "0.7.4",
+    "hls.js": "0.7.11",
```

Comment:
Shouldn't this be in a separate commit? I'd say this is something we want in the changelog.